### PR TITLE
Stop getting iFrame dups and only inject into Iframes if including iframes

### PR DESCRIPTION
**Pull request by TroyWalshProf**

### CODE REVIEW

File: Selenium.Axe/Selenium.Axe/HtmlReport.cs [hunk 1]
```diff
@@ -420,7 +413,7 @@ private static void GetReadableAxeResults(AxeResultItem[] results, ResultType ty
 
                     foreach (var target in item.Target)
                     {
-                        content.AppendLine($"{HttpUtility.HtmlEncode(target)}");
+                        content.AppendLine($"{HttpUtility.HtmlEncode(target.ToString().Replace(",", " "))}");
```
Comment (dbjorge) [inline Selenium.Axe/Selenium.Axe/HtmlReport.cs:416]: Is this change intended to be included? It doesn't feel related to the PR title.

It feels a bit dangerous to me; it's possible for an individual target to be, for example, `div[aria-label="foo, bar, and baz"]`

**Review by dbjorge — CHANGES_REQUESTED** [review]
The part of the change affecting the iframe injection looks good to me, but had one question about an unrelated-looking part of the changes

File: Selenium.Axe/Selenium.Axe/HtmlReport.cs (hunk 1 above)
Comment (TroyWalshProf) [reply Selenium.Axe/Selenium.Axe/HtmlReport.cs:416]: Oddly it is related.
The way the formatting works means if we don't do this it will result in an invalid CSS Selector.
This change makes the HTML report match the Axe extension results.

Comment (TroyWalshProf) [reply Selenium.Axe/Selenium.Axe/HtmlReport.cs]: Selector doesn't really work as is so reverting
